mlflow/tracing/export/mlflow.py

```diff
@@ -80,3 +80,8 @@ def _log_trace(self, trace: Trace):
             self._client._upload_ended_trace_info(trace.info)
         except Exception as e:
             _logger.debug(f"Failed to log trace to MLflow backend: {e}", exc_info=True)
+
+        try:
+            self._client._upload_trace_spans_as_tag(trace.info, trace.data)
```

Comment:
Can we put this above end_trace? It seems more reasonable that we update tags before ending trace (which includes status).

Reply:
I think that's fair